### PR TITLE
Spectra v2.7.2

### PR DESCRIPTION
**Pull request by adi3890**
* Fix: Display Conditions - Resolved an issue wherein the `UAGDay` attribute would persist in the code view after turning off display conditions.
* Fix: Social Share - Converted anchors to span tags and removed unwanted attributes from the social links.
* Fix: Forms - Resolved an issue in which some phone country-code colors would not work as intended in the editor.
* Fix: Table Of Content - Resolved an issue where default padding of Table of Content is not same in editor as in frontend.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -215,6 +215,12 @@ Our external packages use [Rating Star Component](https://github.com/n49/react-s
 
 ## Changelog ##
 
+### x.x.x ###
+* Fix: Display Conditions - Resolved an issue wherein the `UAGDay` attribute would persist in the code view after turning off display conditions.
+* Fix: Social Share - Converted anchors to span tags and removed unwanted attributes from the social links.
+* Fix: Forms - Resolved an issue in which some phone country-code colors would not work as intended in the editor.
+* Fix: Table Of Content - Resolved an issue where default padding of Table of Content is not same in editor as in frontend.
```
Comment (bsf-zanev) [inline README.md:222]: Reorder required, with some minor improvements.

```suggestion
* Fix: Forms - Resolved an issue in which some phone country-code colors would not work as intended in the editor.
* Fix: Table Of Contents - Resolved an issue wherein the default padding not same in the editor and on the frontend.
* Fix: Social Share - Converted anchor tags to span tags and removed unwanted attributes from the social links.
* Fix: Display Conditions - Resolved an issue wherein the `UAGDay` attribute would persist in the code view after turning off display conditions.
```

File: src/blocks/social-share-child/deprecated.js [hunk 2]
```diff
@@ -7,6 +7,7 @@ import classnames from 'classnames';
 import attributes from './attributes';
 import renderSVG from '@Controls/deprecatedRenderIcon';
 import links from './deprecatedLinks';
+import V2_6_9Save from './deprecated/V2_6_9/save';
```
[the remark this answers is not in the document]
Comment (AbhijitNage123) [reply src/blocks/social-share-child/deprecated.js]: Resolved

**Review by AbhijitNage123 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (rajmohanreddy774) [reply README.md:222]: Done.

File: src/blocks/extensions/advanced-settings/index.js [hunk 3]
```diff
@@ -17,14 +18,14 @@ const UserConditionOptions = ( props ) => {
...
 		const { value, checked } = e.target;
-
+		let setUAGBDay;
 		if ( checked ) {
-			setAttributes( { UAGDay: [ ...UAGDay, value ] } );
+			setUAGBDay = { UAGDay: [...UAGDay, value] };
 		} else {
-			setAttributes( { UAGDay: UAGDay.filter( ( i ) => i !== value ) } );
+			setUAGBDay = { UAGDay: updateUAGDay( UAGDay, value ) };
 		}
```
Comment (bsf-zanev) [inline src/blocks/extensions/advanced-settings/index.js:27]: Not a blocker for release, but a suggestion.

```
const setUAGBDay = checked ? { UAGDay: [...UAGDay, value] } : { UAGDay: updateUAGDay( UAGDay, value ) };
```

File: src/blocks/extensions/condition-block/index.js [hunk 4]
```diff
@@ -21,14 +22,14 @@ const UserConditionOptions = ( props ) => {
...
 		const { value, checked } = e.target;
-
+		let setUAGBDay;
 		if ( checked ) {
-			setAttributes( { UAGDay: [ ...UAGDay, value ] } );
+			setUAGBDay = { UAGDay: [...UAGDay, value] };
 		} else {
-			setAttributes( { UAGDay: UAGDay.filter( ( i ) => i !== value ) } );
+			setUAGBDay = { UAGDay: updateUAGDay( UAGDay, value ) };
 		}
```
Comment (bsf-zanev) [inline src/blocks/extensions/condition-block/index.js:31]: Same suggestion here.

**Review by bsf-zanev — COMMENTED** [review]
Just a minor doubt, and few suggestions for the future.

Also, in JS we should start implementing `a = b || c;` instead of `a = b ? b : c` when setting defaults.

https://www.phind.com/agent?cache=clkl65tm3003olg08dgg4fg4o

File: src/blocks/extensions/condition-block/index.js (hunk 4 above)
Comment (bsf-zanev) [reply src/blocks/extensions/condition-block/index.js:31]: Accidently deleted the previous suggestion.

In both files that we're using the `updateUAGDay()` function, we can assign the value in a ternary.

`const  setUAGBDay = checked ? { UAGDay: [...UAGDay, value] } : { UAGDay: updateUAGDay( UAGDay, value ) };`

Comment (adi3890) [reply src/blocks/extensions/condition-block/index.js]: @bsf-zanev @AbhijitNage123 is this resolved in https://github.com/brainstormforce/ultimate-addons-for-gutenberg/pull/3369.
If so please resolve the conversation

Comment (bsf-zanev) [reply src/blocks/extensions/condition-block/index.js]: This has been updated in [this commit](https://github.com/brainstormforce/ultimate-addons-for-gutenberg/pull/3365/commits/2db98c9caa07c2bc56313c1ad297f20f5d40261f).

**Review by bsf-zanev — APPROVED** [review]

**Review by aapatre — APPROVED** [review]

**Review by rahulvarma722 — APPROVED** [review]

**Pull request merged**